README

```diff
@@ -6,8 +6,8 @@ communicate with the Maliit server via input context plugins. The communication
 link currently uses D-Bus. Maliit is an open source framework (LGPL 2) with
 open source plugins (BSD).
 
-This is the Maliit framework. For plugins, such as virtual keyboards, see
-http://wiki.maliit.org/Plugins
+This is the Maliit framework. For a virtual keyboards, see
```

Comment:
this " a virtual keyboards" reads wrong.

Reply:
Oops. Indeed. Fixed.